src/Web/WebClient.php

```diff
@@ -395,19 +395,19 @@ protected function detectEngine($userAgent)
             $this->engine = self::BLINK;
         } elseif (\stripos($userAgent, 'Chrome') !== false) {
             $result  = \explode('/', \stristr($userAgent, 'Chrome'));
-            $version = \explode(' ', $result[1]);
+            $version = \count($result) > 1 ? \explode(' ', $result[1]) : false;
```

Comment:
A simpler fix?
```
if (isset($result[1])) {
}
```

Reply:
Would also work. But is that really simpler?

Reply:
Less code to read/process..one test vs a count and an extra test for false.
```
            $result  = \explode('/', \stristr($userAgent, 'Chrome'));
            if (isset($result[1])) {
            	$version = \explode(' ', $result[1]);

            	if ($version[0] >= 28) {
                	$this->engine = self::BLINK;
            	} else {
                	$this->engine = self::WEBKIT;
            	}
            }
```

Reply:
But this code behaves different to what I proposed. In your case $this->engine will not be set at all when there was no version.

Reply:
In my proposal it is set to BLINK. Not sure what is right, but it’s different.

Reply:
I‘ve tried to make it similar to other checks for versions a few lines below.

Reply:
But maybe your proposal is right and there should not be any engine detected when there is no version. I have to investigate that, or maybe someone can advise.

Reply:
You are absolutely right. 👍 Please ignore my noise.

Maybe this?
```
$version = \count($result) > 1 ? \explode(' ', $result[1]) : false;
$version = isset($result[1]) ? \explode(' ', $result[1]) : false;
```

Reply:
Yes, that would simplify my code. Will apply it in a minute. But still need to investigate if my fix is right. It may be next weekend until I can do that.

Reply:
It feels off to me too.